smart-lock

```diff
@@ -236,7 +236,7 @@ function pi-compile {
     match-keyword 'compile'
     expected-end-of-input
 
-    ${DOCKER_COMMAND} run --rm -it -v "${_SMART_LOCK_DIR}/src:/home/pi/smartlock" -w "/home/pi/smartlock" pi-opendds-cross bash -c "source /home/pi/pi-opendds/build/target/setenv.sh; sudo chown -R pi .; mwc.pl -type gnuace -features 'no_pigpio=0'; make"
+    MSYS_NO_PATHCONV=1 ${DOCKER_COMMAND} run --rm -it -v "${_SMART_LOCK_DIR}/src:/home/pi/smartlock" -w "/home/pi/smartlock" pi-opendds-cross bash -c "source /home/pi/pi-opendds/build/target/setenv.sh; sudo chown -R pi .; mwc.pl -type gnuace -features 'no_pigpio=0'; make"
```

Comment:
Would it be easier to set this once in the environment so all child processes inherit it?  Or does it need to be more selectively enforced?

Reply:
It must be selectively chosen.  If it is set for the entire file, other paths that need to be converted are not.